composer.json

```
@@ -119,6 +119,7 @@
         "psr/simple-cache-implementation": "1.0|2.0|3.0"
     },
     "conflict": {
+        "carbonphp/carbon-doctrine-types": "^3.0",
```

@timacdonald: @kylekatarnls gave the good remark that adding `doctrine/dbal` v4 here might be a better solution. We could even add them both. This also saves us from having to update the docs.
```suggestion
        "carbonphp/carbon-doctrine-types": ">=3.0",
        "doctrine/dbal": ">=4.0",
```